src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -61,18 +65,41 @@ export default function NftDrawer({
     savedCollectionSymbol,
   )
 
+  const defaultCollectionName = `${projectName} NFT rewards`
+  const defaultCollectionDescription = `The NFT rewards of the Juicebox project ${projectName}.`
```

Comment:
Will add some more info about each tier in this default value in a follow-up (#1915)

Reply:
```suggestion
  const defaultCollectionDescription = `${projectName}'s Juicebox NFT rewards.`
``` maybe, idk

Reply:
```suggestion
  const defaultCollectionDescription = `The NFT's rewarded to ${projectName}'s Juicebox contributors.`
``` or something